src/playbooks/setup-hammer/setup-hammer.yaml

```diff
@@ -8,5 +8,6 @@
     - "../../vars/{{ certificate_source }}_certificates.yml"
   vars:
     hammer_ca_certificate: "{{ server_ca_certificate }}"
+    hammer_plugins: "{{ plugins | map('replace', 'foreman-tasks', 'foreman_tasks') }}"
```

Comment:
Everything is lower case, except `foreman-tasks`. Perhaps we should from a foremanctl perspective just consider it lower case and then special case it where needed?

Perhaps even already fix https://github.com/theforeman/foreman-oci-images/blob/220306c20253c7bbb3ae223fbdf5656ab7887c65/images/foreman/Containerfile#L17-L19 before we merge this.